pandas/core/indexes/multi.py

```diff
@@ -2328,23 +2337,40 @@ def convert_indexer(start, stop, step, indexer=indexer, labels=labels):
             j = labels.searchsorted(loc, side='right')
             return slice(i, j)
 
-    def get_locs(self, tup):
+    def get_locs(self, seq):
         """
-        Given a tuple of slices/lists/labels/boolean indexer to a level-wise
-        spec produce an indexer to extract those locations
+        Get location for a given label/slice/list/mask or a sequence of such as
+        a integer, slice or boolean mask.
 
         Parameters
         ----------
-        key : tuple of (slices/list/labels)
+        seq : label/slice/list/mask or a sequence of such.
```

Comment:
It might be good to explain how the 'sequence of such' should be interpreted

(I should check the code myself to actually know it, but are the different elements of the list mapped to the different levels?)

Reply:
maybe add here "(one for each level)" as you did in get_loc

Reply:
I've made a new proposal. This is very difficult to explain in short sentences, I find.